components/frontend/src/locale.js

```diff
@@ -0,0 +1,30 @@
+import "dayjs/locale/en-gb"
+import "dayjs/locale/nl"
+
+import dayjs from "dayjs"
+import localeData from "dayjs/plugin/localeData"
+import updateLocale from "dayjs/plugin/updateLocale"
+import { string } from "prop-types"
+
+export function adapterLocale(language) {
+    // Quick and dirty solution while we wait for a more generic solution to finding the user's locale
```

Comment:
Do we have a tracking issue to solve this ourselves?

Reply:
Good point. I'll open an issue.